CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+## How to contribute to guides.rubygems.org
+
+#### **If you want to contribute to RubyGems itself, visit [the RubyGems Guides](http://guides.rubygems.org/contributing/)**
```

Comment:
This is pretty confusing as to what's happening here, either remove the `**` or don't make this a header